lib/collective-sections.js

```diff
@@ -225,6 +225,10 @@ const getSectionsToRemoveForUser = (collective, isAdmin) => {
     toRemove.add(Sections.CONTRIBUTORS);
   }
 
+  if (features[FEATURES.TRANSACTIONS] !== 'ACTIVE') {
```

Comment:
What about cases where a collective has no transaction but pending expenses? To be tested, but I think we want something like

```suggestion
  if (features[FEATURES.TRANSACTIONS] !== 'ACTIVE' && features[FEATURES.RECEIVE_EXPENSES] !== 'ACTIVE') {
```

Reply:
Thanks for pointing it out. I've added this and indeed we need this in case of pending expenses. 👍🏽